src/schema.ts

```diff
@@ -198,6 +198,35 @@ export class Schema {
     }
   }
 
+  /**
+   * Given an EncodingQuery with a timeUnit, returns true if the number of
```

Comment:
This reads quite confusing.  

> returns true if the number of
 dates that are completely distinct in all parts of the timeUnit is greater than 1.

Maybe

> returns true if the date field has multiple distinct values for all parts of the timeUnit?